dcm4che-net-audit/src/test/java/org/dcm4che3/net/audit/ConfigurablePropertiesTest.java

```diff
@@ -55,7 +55,10 @@ protected Class[] getClassesToTest() {
     }
 
     @Override
-    protected Set<Field> getPropertiesToSkip() {
-        return new HashSet<>();
+    protected Set<Field> getPropertiesToSkip() throws NoSuchFieldException {
```

Comment:
Not sure why this is showing up in your PR as you didn't touch those classes? Also my class looks different, I'm confused.

https://github.com/msiemczyk/dcm4che/blob/generic-config/dcm4che-net/src/test/java/org/dcm4che3/net/ConfigurablePropertiesTest.java

Reply:
I needed to add AuditLoggerTest.java in the test directory but noticed that there was an existing ConfigurablePropertiesTest  file that wasn't in the correct test directory and hence was never run.  When I moved ConfigurablePropertiesTest file into the proper directory, the test failed and hence the changes.

Reply:
Ah, I didn't see that the file was move. Maybe it would make more sense to combine both files then? i.e. just move the classes and Fields to skip into "net/ConfigurablePropertiesTest.java"?